{{cookiecutter.project_slug}}/dev.yml

```diff
@@ -19,7 +19,8 @@ services:
       dockerfile: ./compose/django/Dockerfile-dev
     command: /start-dev.sh
     depends_on:
-      - postgres
+      - postgres{% if cookiecutter.use_mailhog == 'y' %}
```

Comment:
jinja2 have a sintax to control newlines/whitespace

http://jinja.pocoo.org/docs/2.9/templates/#whitespace-control